base/applications/charmap/charmap.c

```diff
@@ -25,6 +25,63 @@ HICON     hSmIcon;
 HICON     hBgIcon;
 SETTINGS  Settings;
 
+static
+VOID
+FillCharacterSetComboList(HWND hwndCombo)
+{
+    WCHAR lpCharSetText[256];
+    WCHAR* trimmedName;
+    CPINFOEXW cpInfo;
+    INT i;
+
+    if (LoadStringW(hInstance, IDS_UNICODE, lpCharSetText, SIZEOF(lpCharSetText)))
+    {
+        SendMessageW(hwndCombo,
+            CB_ADDSTRING,
```

Comment:
Please use alignment of parameters under the previous ones, like in:
```c
SendMessageW(hwndCombo,
             CB_ADDSTRING,
             0,
             (LPARAM)lpCharSetText);
```
(same comment for the other parts of code too.)